src/math/trig.c

```diff
@@ -32,17 +24,31 @@ const int32_t cordic_sine_cos_piovertwo_q30fl  = Q_CONVERT_FLOAT(_M_PI / 2, 30);
 const int32_t cord_sincos_piovertwo_q28fl  = Q_CONVERT_FLOAT(_M_PI / 2, 28);
 /* 843314857,  deg = 90.000000	*/
 const int32_t cord_sincos_piovertwo_q29fl  = Q_CONVERT_FLOAT(_M_PI / 2, 29);
+/* arc trignometry constant*/
+/**
+ * \+----------+----------------------------------------+--------------------+-------------------+
+ * \|values    |		Q_CONVERT_FLOAT		|(180/pi)*angleInRad |(pi/180)*angleInDeg|
+ * \+----------+----------------------------------------+--------------------+-------------------+
+ * \|379625062 | Q_CONVERT_FLOAT(1.4142135605216026, 28)|    81.0284683480568| 1.41421356052160  |
+ * \|1073741824| Q_CONVERT_FLOAT(1.0000000000000000, 30)|    57.2957795130823| 1.00000000000000  |
+ * \|843314856 | Q_CONVERT_FLOAT(1.5707963258028030, 29)|    89.9999999431572| 1.57079632580280  |
+ * \|1686629713| Q_CONVERT_FLOAT(1.5707963267341256, 30)|    89.9999999965181| 1.57079632673413  |
+ * \+----------+----------------------------------------+--------------------+-------------------+
+ */
+/* 379625062,  deg = 81.0284683480568475 or round(1.4142135605216026*2^28) */
+const int32_t cord_arcsincos_q28fl  = Q_CONVERT_FLOAT(1.4142135605216026 / 2, 28);
```

Comment:
I found that the comment here confuses me. The code is Q_CONVERT_FLOAT(1.4142135605216026 / 2, 28).
However, the comment is Q_CONVERT_FLOAT(1.4142135605216026, 28), round(1.4142135605216026*2^28)

Which one is your real usage?

Reply:
real one is `const int32_t cord_arcsincos_q28fl  = Q_CONVERT_FLOAT(1.4142135605216026 / 2, 28);`

I am not 100% sure 
`Q_CONVERT_FLOAT(1.4142135605216026, 28), round(1.4142135605216026*2^28)` exist 
but `round(1.4142135605216026*2^28)` is kind of cross check/reference ( Matlab/Octave ) since Q_CONVERT_FLOAT is 100% guarantee 

In case if this is not ok, I can remove matlab/octave comments from code .

Reply:
round(1.4142135605216026*2^28) is Q_CONVERT_FLOAT(1.4142135605216026, 28), so if it is used in your Matlab checking then it explains all.
saturated_unary_minus is Q2.30, Q2.30 --( >>1 )--> Q3.29, and it compares to Q_CONVERT_FLOAT(1.4142135605216026, 28)

And Q_CONVERT_FLOAT(1.4142135605216026, 28) equals to Q_CONVERT_FLOAT(1.4142135605216026 / 2, 29)

So it just matched!!!

Reply:
Thanks ,Yup - need to careful with Q_CONVERT_FLOAT !!